### DISCUSSION TITLE
如何调整主页布局？

### DISCUSSION THREAD

**Discussion by JinchaoLove** [vuepress-theme-hope/vuepress-theme-hope#3260] [Q&A] [answered]
如图，我想要把`InfoPanel`里面的“文章——时间轴”和“medias”位置互换，如何实现呢？

<img width="283" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/34153355/a0ae2837-b746-4543-b384-5e713d95f12f">

**Comment by Mister-Hope** [comment, marked as answer]
替换组件

**Reply by JinchaoLove** [reply]
感谢大佬！请问是`BloggerInfo.js`文件吗？我目前替换的是`node_modules/vuepress-theme-hope/lib/client/modules/blog/components/BloggerInfo.*`，包括`.ts`, `.js`和`.js.map`，但感觉不太优雅😂

**Reply by Mister-Hope** [reply]
需要别人教你去文档搜替换组件，然后看文档吗？

**Reply by JinchaoLove** [reply]
额，我看过文档了https://theme-hope.vuejs.press/zh/guide/advanced/replace.html，替换的是@theme-hope/modules/blog/components/BloggerInfo.*三个文件，也成功了。只是好奇是否有其他的方法。
Anyway，感谢🙏

**Reply by Mister-Hope** [reply]
如果你真的看懂了文档，你不会问我为什么要替换js，为什么要替换js.map？

**Reply by JinchaoLove** [reply]
确实没怎么看懂，以前用的jekll，今天刚接触这个。我先用段时间应该就明白了～感谢～
